hooks/pre-command

```diff
@@ -41,15 +41,20 @@ else
     exit 1
 fi
 
-if [[ "$version" == "nightly" ]]; then
+if [[ "$version" == "nightly" ]]; then        # `version: 'nightly'`
     release="nightly"
     filename="julia-latest-$os$nightly_rarch.tar.gz"
     url="https://julialangnightlies-s3.julialang.org/bin/$os/$larch/$filename"
-elif [[ "$version" =~ ^(.+)-nightly$ ]]; then
+elif [[ "$version" =~ ^(.+)-nightly$ ]]; then # for example, `version: '1.6-nightly'`
     release=${BASH_REMATCH[1]}
     filename="julia-latest-$os$nightly_rarch.tar.gz"
     url="https://julialangnightlies-s3.julialang.org/bin/$os/$larch/$release/$filename"
-else
+elif [[ $version =~ ^-?[0-9]+$ ]]; then       # for example, `version: '1'` or `version: '234'`
+    parentdirectory=$(dirname $BASH_SOURCE)
+    release=$(python3.8 $parentdirectory/expand-major-only.py $version)
```

Comment:
Is the script `python3` specific? If so, use `python3`, if not just `python`; it shouldn't be tied to our specific Docker image.

Reply:
So, I tried so desperately to get that to work. I could not get either `python` or `python3` to work. It kept giving me "command not found". 

The only way that I could get it to work was to use `python3.8`.

Reply:
Because you install `python3.8`. If you install `python3` (which depends on `python3.8`), it should be available. See https://packages.ubuntu.com/search?searchon=contents&keywords=python3&mode=exactfilename&suite=groovy&arch=any

Reply:
🤦

Reply:
Okay, should be fixed now:
1. In the Docker image, I install `python3` (instead of `python3.8`)
2. In this script, I call the `python3` command
3. In the requirements, we list `python3` and `wget`.